engine/src/flutter/lib/web_ui/test/engine/text_editing_test.dart

```diff
@@ -1545,6 +1545,39 @@ Future<void> testMain() async {
       hideKeyboard();
     });
 
+    test('Words capitalization: setClient, setEditingState, show', () {
+      // Create a configuration with an AutofillGroup of four text fields.
+      final Map<String, dynamic> capitalizeWordsConfig = createFlutterConfig(
+        'text',
+        textCapitalization: 'TextCapitalization.words',
+      );
+      final MethodCall setClient = MethodCall('TextInput.setClient', <dynamic>[
+        123,
+        capitalizeWordsConfig,
+      ]);
+      sendFrameworkMessage(codec.encodeMethodCall(setClient));
+
+      const MethodCall setEditingState1 = MethodCall('TextInput.setEditingState', <String, dynamic>{
+        'text': '',
+        'selectionBase': 0,
+        'selectionExtent': 0,
+      });
+      sendFrameworkMessage(codec.encodeMethodCall(setEditingState1));
+
+      const MethodCall show = MethodCall('TextInput.show');
+      sendFrameworkMessage(codec.encodeMethodCall(show));
+      spy.messages.clear();
+
+      // Test for mobile Safari.
+      if (ui_web.browser.browserEngine == ui_web.BrowserEngine.webkit &&
+          ui_web.browser.operatingSystem == ui_web.OperatingSystem.iOs) {
+        expect(textEditing!.strategy.domElement!.getAttribute('autocapitalize'), 'words');
+      }
```

Comment:
Since this is supposed to work in all browsers now, let's test all browsers:
```suggestion
      expect(textEditing!.strategy.domElement!.getAttribute('autocapitalize'), 'words');
```

Reply:
Oh my bad. The `autocapitalize` attribute is only applied for mobile browsers:
https://github.com/flutter/flutter/blob/e6c333b555b5be4d339ed27d24a364c2719993d4/engine/src/flutter/lib/web_ui/lib/src/engine/text_editing/text_editing.dart#L1662
https://github.com/flutter/flutter/blob/e6c333b555b5be4d339ed27d24a364c2719993d4/engine/src/flutter/lib/web_ui/lib/src/engine/text_editing/text_editing.dart#L1792